core/src/test/scala/org/apache/pekko/persistence/r2dbc/journal/RuntimePluginConfigSpec.scala

```diff
@@ -0,0 +1,312 @@
+/*
```

Comment:
new files that are not copies of old Akka files should be given the standard Apache header not the Akka one